lib/ui/views/delete_account/delete_account_view.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../../locator.dart';
+import '../../../services/auth.dart';
+import '../../../services/user_service.dart';
+import 'delete_account_bloc.dart';
+import 'delete_account_state.dart';
+
+@RoutePage()
+class DeleteAccountView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (_) => DeleteAccountBloc(
+        authenticationService: locator<AuthenticationService>(),
+        appRouter: AutoRouter.of(context),
+        userService: locator<UserService>(),
+      ),
+      child: BlocBuilder<DeleteAccountBloc, DeleteAccountState>(
```

Comment:
More of a question, but is it generally better to have the bloc builders lower down in the tree (e.g. only wrapping the submit button and input) or is this ok? I assume the latter is better as it results in less re rendering but basically irrelevant for a page as simple as this but more generally is a good idea

Reply:
Yeah I recon so, I'm used to Android Compose way where only changed things are rebuilt, fixed thanks!